packages/react/src/indicator-dot/docs/overview.mdx

```diff
@@ -28,7 +28,6 @@ An Indicator dot should not be used on its own, as it gives no context. It shoul
 <Flex flexDirection="column" as="ul">
 	<Flex
 		as="li"
-		aria-selected="false"
```

Comment:
Could we remove this from the stories too please? `packages/react/src/indicator-dot/IndicatorDot.stories.tsx`